### PR TITLE
Added sonarqube scanning

### PR DESCRIPTION
**Pull request by localgod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
 # Kumori
 
 Healthcheck of your architecture.
 
-This projects uses [Browserstack](https://www.browserstack.com) for automated testing. 
-
-
-
-
+This projects uses [Browserstack](https://www.browserstack.com) for automated testing.
```
Comment (themathmagician) [inline README.md]: I am missing a description of the purpose of the tools (what problem ...) for sonarqube and browserstack
what do they do for us - any usage policies?

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
+sonar.projectKey=Mathmagicians_kumori
+sonar.organization=mathmagicians
+sonar.projectName=Kumori
+sonar.projectVersion=1.0.0
+sonar.sources=meritocracy/frontend/src
+sonar.tests=meritocracy/frontend/test
```
Comment (themathmagician) [inline sonar-project.properties]: vi har også tests i Spec og Backend?

File: README.md (hunk 1 above)
Comment (localgod) [reply README.md]: This reference is added as a request from browserstack in exchange for supporting the project with a free usage of there service. I have yet to set the test up.

File: sonar-project.properties (hunk 2 above)
Comment (localgod) [reply sonar-project.properties]: First make it work....(this version)...

**Pull request merged**